_overviews/scaladoc/for-library-authors.md

```diff
@@ -239,6 +239,62 @@ The markup for list blocks looks like:
 - DRY - don't repeat yourself. Resist duplicating the method description in the
   `@return` tag and other forms of repetitive commenting.
 
+## Resolving Ambiguous Links within Scaladoc Comments
+When two methods are indistinguishable from each other lexically, it can cause Scaladoc to
+report that there are ambiguous methods. As an example:
+
+```scala
+import scala.collection.mutable.ListBuffer
+class bar {
+    def foo(x: Int): Boolean = ???
+    def foo(x: ListBuffer[Int], y: String): Int = ???
+}
+```
+
+If one references `foo` via `[[foo]]`, then the Scaladoc will complain and offers up both
+alternatives. Fixing this means elaborating the signature _enough_ so that it becomes unambiguous.
+There are a few things to be aware of in general:
+
+* You must not use a space in the description of the signature: this will cause Scaladoc to
+  think the link has ended and move onto its description.
+* You must fully qualify any types you are using: assume that you have written your program without
+  any import statements!
+
+Then, to disambuate between objects and types (where both are in use), postfix with a `$` for objects
```

Comment:
disambiguate

terms and types. ~(where both are in use)~ is redundant.

"append `$` to designate a term name" etc.

If you think it necessary: "Term names include members which are not types, such as `val`, `def`, and `object` definitions." Pairings such as `class C(val c: Int); def C(c: Int) = new C(c)` would be prone, IIUC.